src/main/java/org/jitsi/videobridge/cc/AdaptiveTrackProjection.java

```diff
@@ -380,19 +381,19 @@ private RtpState getRtpState()
     /**
      * Rewrites an RTCP packet.
      *
-     * @param rtcpPacket the RTCP packet to rewrite.
+     * @param rtcpSrPacket the RTCP SR packet to rewrite.
      * @return true to let the RTCP packet through, false to drop.
      * XXX unused???
      */
-    public boolean rewriteRtcp(@NotNull RawPacket rtcpPacket)
+    public boolean rewriteRtcp(@NotNull RtcpSrPacket rtcpSrPacket)
```

Comment:
are RTCP SRs the only packets we'll ever send here?  if so, should we name it as such (since we're no longer bound to a generic rewriteRtp/rewriteRtcp interface)?  if not (and we'll send other rtcp through here) then i think we should leave the generic `rtcpPacket` param name and test for whether or not it's an sr in the method

Reply:
eventually we would like to bring back the old processing for bye and sdes, so this code would have a compound rtcp packet as a parameter (there's some details about that in the trello board); but sdes/bye handling is not guarantee to make its way in this pr. i don't particularly like rewriteRcp either